### PR TITLE
[Insult] 57 more insults <:smirk:728038483251429476>

### PR DESCRIPTION
**Pull request by owocado**
I have double-checked these with existing insults, they're unique. Please let me know what do you think.

~~Also want to experiment if github webhook will render the smirk emoji on your discord channel or not when merged. Hope you won't mind.~~

### CODE REVIEW

File: insult/insult.py [hunk 1]
```diff
@@ -260,6 +260,63 @@
...
+	_("You sound reasonable. It must be time to up my medication!"),
+	_("If I had a face like yours, I'd sue my parents."),
+	_("There's only one problem with your face. I can see it."),
+	_("Don't you love nature, despite what it did to you?"),
+	_("What language are you speaking? Cause it sounds like bullshit."),
+	_("You have a room temperature IQ - if the room is in Antarctica."),
+	_("I would ask you how old you are but I know you can't count that high."),
+	_("Do you want to know how I get all these insults? I use something called intelligence."),
+	_("I was going to give you a nasty look, but you already have one."),
+	_("Brains aren't everything. In your case they're nothing."),
```
Comment (TrustyJAID) [inline insult/insult.py]: This insult is repeated in the list already. "Brains aren't everything. In your case they're nothing."

File: insult/insult.py [hunk 2]
```diff
@@ -260,6 +260,63 @@
     ),
     _("Your father could've used a condom but here you are."),
     _("You look like a randomized sim."),
+    _("*Puts you in the oven.*"),
+	_("You're so stupid."),
+	_("Sorry, I can't hear you over how annoying you are."),
+	_("I've got better things to do."),
+	_("You're as dumb as Cleverbot."),
+	_("Your IQ is lower than the Mariana Trench."),
```
Comment (TrustyJAID) [inline insult/insult.py]: It's "Marianas Trench"

File: insult/insult.py (hunk 1 above)
Comment (owocado) [reply insult/insult.py]: OMG how did I miss this? Sincere apologies. Pushed the fix.

File: insult/insult.py (hunk 2 above)
Comment (owocado) [reply insult/insult.py]: I think both "The Mariana Trench" and "Marianas Trench" are valid.

https://i.imgur.com/WrWAW2a.png
Source: https://en.wikipedia.org/wiki/Mariana_Trench

what do you think?

**Comment by NeuroAssassin** [comment]
image

**Pull request merged**